krit/registration/serializers.py

```diff
@@ -6,10 +6,19 @@
 UserModel = get_user_model()
 
 
+def _get_user_registration_fields():
+    """Return fields to serialize from UserModel"""
+    if type(UserModel.__dict__.get('registration_fields')) == tuple:
```

Comment:
DRF [takes a list too](http://www.cdrf.co/3.7/rest_framework.serializers/ModelSerializer.html#get_field_names). Not a big deal though.

Reply:
Eh, not a big deal to allow lists either. I'll allow those too